support/cas-server-support-pac4j-webflow/src/main/java/org/apereo/cas/web/flow/DelegatedClientAuthenticationAction.java

```diff
@@ -243,6 +243,10 @@ protected Event handleException(final J2EContext webContext, final BaseClient<Cr
         }
     }
 
+    private boolean isLogoutRequest(final HttpServletRequest request) {
```

Comment:
This method should be used line 456 as well.

Reply:
Updated line 456.